src/main/java/com/cloudbees/jenkins/GitHubWebHook.java

```diff
@@ -204,13 +208,20 @@ public void processGitHubPayload(String payload, Class<? extends Trigger<?>> tri
             Authentication old = SecurityContextHolder.getContext().getAuthentication();
             SecurityContextHolder.getContext().setAuthentication(ACL.SYSTEM);
             try {
-                for (AbstractProject<?, ?> job : Jenkins.getInstance().getAllItems(AbstractProject.class)) {
-                    GitHubTrigger trigger = (GitHubTrigger) job.getTrigger(triggerClass);
-                    if (trigger != null) {
+                for (Job<?, ?> job : Jenkins.getInstance().getAllItems(Job.class)) {
+                    GitHubTrigger ghTrigger = null;
+                    ParameterizedJobMixIn.ParameterizedJob pJob = (ParameterizedJobMixIn.ParameterizedJob) job;
+                    for (Trigger trigger : pJob.getTriggers().values()) {
+                        if (triggerClass.isInstance(trigger)) {
```

Comment:
Yes, it's what I need, but the right operand (usually ```GitHubTrigger```, but not always) is given as a method parameter, so can not use ```instanceof```.